gammapy/cube/counts.py

```diff
@@ -55,7 +55,7 @@ def fill_map_counts(counts_map, events):
     cols = {k.upper(): v for k, v in events.table.columns.items()}
 
     for axis in counts_map.geom.axes:
-        if axis.type == 'energy':
+        if axis.name.lower() in ['energy', 'energy_reco']:
```

Comment:
Why sometimes `.upper` and sometimes `.lower`?
Pick one and use everywhere (at least in this function)?

Reply:
OK